test/proto.js

```diff
@@ -735,6 +735,36 @@ describe('proto', function(){
       assert(false == segment.retry(new TypeError('whoops')));
     });
   });
+
+  describe('.enabled()', function(){
+    var Test = integration('test').server();
+    var test = Test({});
+
+    it('should return true when sent on a supported channel', function(){
+      var facade = new Page({ channel: 'server' });
+      assert(true === test.enabled(facade));
```

Comment:
In general, prefer `assert.strictEquals(x, y)` over `assert(x === y)`. It gives you way more information when the test fails